pkg/odo/genericclioptions/clientset/clientset.go

```diff
@@ -0,0 +1,111 @@
+// clientset package is used to inject clients inside commands
```

Comment:
This is probably not that important. But when I was browsing through the code on GoLand, it suggested some fixes. Attaching a screenshot. The changes can be found on https://github.com/dharmit/odo/commit/c3e5429304f2e72e0f0e50e8505801a95b3186d7
![Screenshot from 2022-02-04 09-29-14](https://user-images.githubusercontent.com/713521/152469774-2dd056a5-bb11-4295-a510-5fb8d40c7ad3.png)